include/bliss.h

```diff
@@ -16,11 +16,11 @@
     #define av_frame_free avcodec_free_frame
 #endif
 
-static const int BL_LOUD = 0;
-static const int BL_CALM = 1;
-static const int BL_UNKNOWN = 2;
-static const int BL_UNEXPECTED = -2;
-static const int BL_OK = 0;
+const int BL_LOUD = 0;
```

Comment:
These shouldn't be in headers, because this will cause linking issues. Instead using an enum/define would be better.

Reply:
fixed, now uses defines rather than constants